### ISSUE TITLE
Question about 'mask_label' in transformer.mam.process_train_MAM_data

### ISSUE THREAD

**Issue by hzlihang99** [andi611/Self-Supervised-Speech-Pretraining-and-Representation-Learning#51]
Hi,

Thank you so much for providing us such well-organized Pretraining framework. 

Recently, I got a question about the variable 'mask_label' defined in your transformer.mam.process_train_MAM_data. 

`mask_label = torch.zeros_like(spec_stacked, dtype=torch.uint8) if mask_proportion != 0 and mask_frequency != 0 else torch.ones_like(spec_stacked, dtype=torch.uint8)`

From the code, I understand that you are targeting at the masked frames and calculating the L1 loss for those frames only. However, I am thinking maybe it is better to modify the line to

`mask_label = torch.zeros_like(spec_stacked, dtype=torch.uint8) if mask_proportion != 0 or mask_frequency != 0 else torch.ones_like(spec_stacked, dtype=torch.uint8)`

, since sometimes we only want to perform either time mask or frequency mask and we hope the 'mask_label' won't always be a ones matrix in such cases.

**Comment by andi611** [comment]
Hi,

Sorry I did not notice this bug!
The line has been updated.
